_contrib/travis-check-png-optimized.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+## Finds newly added or modifyed PNG files in the repository and checks if each file is optimized.
+## If not optimized it fails with a error code 1
+
+CHANGEDFILES=$(git diff --name-only --diff-filter=AM $TRAVIS_COMMIT_RANGE)
```

Comment:
Nit: Consider running `shellcheck` on this script and address nits :-)

Reply:
Maybe instead of `$TRAVIS_COMMIT_RANGE`, use `$(git merge-base master HEAD)` as something that's likely to be available to any of us?

Reply:
Thanks! Didn't know about `shellcheck`. Ran `spellcheck` on it as well :-)